components/usage/pkg/stripe/stripe.go

```diff
@@ -23,8 +23,9 @@ import (
 )
 
 const (
-	AttributionIDMetadataKey     = "attributionId"
-	PreferredCurrencyMetadataKey = "preferredCurrency"
+	AttributionIDMetadataKey        = "attributionId"
+	PreferredCurrencyMetadataKey    = "preferredCurrency"
+	BillingCreaterUserIDMetadataKey = "billingCreatorUserId"
```

Comment:
Would be great to have a comment here what we use these for :cloud:

Reply:
There are comments in the respective usages. I'll add something..